src/Destructurama.Attributed/Attributed/CustomPropertyInfoExtension.cs

```diff
@@ -0,0 +1,28 @@
+using System.Reflection;
+
+namespace Destructurama.Attributed;
+
+internal static class CustomPropertyInfoExtension
+{
+    /// <summary>
+    /// Returns a list of custom attributes from the specified member or from corresponding member from another
+    /// class if System.ComponentModel.DataAnnotations.MetadataTypeAttribute is used.
+    /// </summary>
+    public static IEnumerable<Attribute> GetCustomAttributesEx(this MemberInfo memberInfo, bool respectMetadata)
+    {
+        if (memberInfo.MemberType != MemberTypes.Property || !respectMetadata)
```

Comment:
Why not just always call `GetCustomAttributesEx` on `PropertyInfo` and remove single call on `type`?

Reply:
Hmm, I thought I changed it on all places, except in extension, because there it will lead to endless loop.

Reply:
How? GetCustomAttributesEx does not call itself.